src/main/plugin/iso19139.ca.HNAP/templates/hnaplevel1-english.xml

```diff
@@ -361,9 +360,8 @@
                               <gco:Date>2004</gco:Date>
                            </gmd:date>
                            <gmd:dateType>
-                              <gmd:CI_DateTypeCode codeList="http://nap.geogratis.gc.ca/metadata/register/napMetadataRegister.xml#IC_87"
+                              <gmd:CI_DateTypeCode codeList="https://schemas.metadata.geo.ca/register/napMetadataRegister.xml#IC_87"
                                                    codeListValue="RI_367">publication; publication</gmd:CI_DateTypeCode>
-                           </gmd:dateType>
```

Comment:
Why is gmd:dateType ending element removed?

Reply:
As above, I do not trust that the conflict resolution was working as intended.

Reply:
As above this is a whitespace change: ``</gmd:dateType>`` looks to have gained some whitespace during conflict resolution.